portfolio/src/main/webapp/authenticationScript.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Shows different messages according to user log in status.
+ */
+function addLoginOrLogoutMessages() {
+  const anchor = document.createElement('a');
+  const initialMessage = document.createElement('span');
+  const linkMessage = document.createElement('span');
+  const container = document.getElementById('login-container');
+  fetch('/login').then(response => response.json()).then((loginData) => {
```

Comment:
nit: Consider at least leaving TODOs for error handling if you're not going to implement them in this PR.

Reply:
How should I handle the errors? What could they be?